content/en/apps/guides/hosting/monitoring/setup.md

```diff
@@ -0,0 +1,168 @@
+
+
+### Prerequisites
+
+- [Docker](https://docs.docker.com/install/)
+- [Docker Compose](https://docs.docker.com/compose/install/)
+- [git](https://git-scm.com/book/en/v2/Getting-Started-Installing-Git)
+- URL(s) of the CHT instance(s)
+
+### Setup
+
+These instructions have been tested against Ubuntu, but should work against any OS that meets the prerequisites. They follow a happy path assuming you need to only set a secure password and specify the URL(s) to monitor:
+
+1. Run the following commands to clone this repository, initialize your `.env` file, create a secure password and create your data directories:
+
+    ```sh
+    cd ~
+    git clone https://github.com/medic/cht-monitoring.git
+    cd cht-monitoring
+    cp cht-instances.example.yml cht-instances.yml
+    cp grafana/grafana.example.ini grafana/grafana.ini
+    mkdir -p grafana/data && mkdir  -p prometheus/data 
+    apt install -y wamerican  # ensures /usr/share/dict/words is present for shuf call below 
+    cp .env.example .env
+    password=$(shuf -n7 /usr/share/dict/words --random-source=/dev/random | tr '\n' '-' | tr -d "'" | cut -d'-' -f1,2,3,4,5,6,7)
+    sed -i -e "s/password/$password/g" .env
+    echo;echo "Initial project structure created! To log into Grafana in the browser:";echo 
+    echo "    username: medic"
+    echo "    password: ${password}";echo
+    ```
+   Note that in step 4 below you'll need the username and password which is printed after you run the above command.
+
+2. Edit the `cht-instances.yml` file to have the URLs of your CHT instances. You may include as many URLs of CHT instances as you like. To preserve data consistency parameters in this file should NOT be adjusted after it is initially set.
+
+   Here is an example:
+
+    ```yml
+    - targets:
+      - https://subsub.sub.example.com
+      - https://cht.domain.com
+      - https://website.org
+    ```
+3. Run the following command to deploy the stack:
+
+    ```sh
+    cd ~/cht-monitoring
+    docker compose up -d
+    ```
+
+4. Grafana is available at [http://localhost:3000](http://localhost:3000). See the output from step 1 for your username and password.
+
+If you would like to do more customizing of your deployment, see ["Addition Configuration"](#additional-configuration).
+
+### Upgrading
+
+Before upgrading, you should back up both your current configuration settings and your Prometheus time-series data.
+
+#### Prometheus, Grafana and JSON Exporter
+
+To upgrade these dependencies, update the version numbers set in your `.env` file (or leave them set to `latest`).  Then run the following commands:
+
+```shell
+docker compose pull
+docker compose up -d --remove-orphans
```

Comment:
```suggestion
docker compose up -d
```

Does not hurt to have the `--remove-orphans` here but might be confusing to a reader since there is no way to get orphan containers from running `docker compose pull`